src/g_i_repository/dumper.cr

```diff
@@ -1,7 +1,7 @@
 module GIRepository
   class Dumper
     macro def
-      def dump(dumper, & : BaseInfo ->)
+      def dump(dumper, &block : BaseInfo ->)
```

Comment:
No need to actually capture the block here, the previous code just restricted the type signature of the block. Maybe you need to update your Crystal version?

Reply:
I am use Crystal 0.30.1 [5e6a1b672] (2019-08-12)

Reply:
Try with 0.31.1 :)